bindata/network-identity/self-hosted/network-identity.yaml

```diff
@@ -0,0 +1,136 @@
+# The network-identity components
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: network-identity
+  namespace: openshift-network-identity
+  annotations:
+    kubernetes.io/description: |
+      This daemonset launches the network-identity networking components.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  selector:
+    matchLabels:
+      app: network-identity
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 100%
+      maxUnavailable: 0
+  template:
+    metadata:
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        app: network-identity
+        component: network
+        type: infra
+        openshift.io/component: network
+        kubernetes.io/os: "linux"
+    spec:
+      serviceAccountName: network-identity
+      hostNetwork: true
+      dnsPolicy: Default
+      priorityClassName: "system-cluster-critical"
+      containers:
+      - name: webhook
```

Comment:
Are we sure we want to run the webhook and the approver on the same pod?

Would we like, for example, to let an administrator restart the webhook without affecting the approver, or viceversa?

Reply:
I think it is fine to combine them just to avoid creating yet another daemonset/deployment.
I don't foresee significant issues if we want to change that approach in the future.